src/components/PositionList.tsx

```diff
@@ -64,7 +64,10 @@ const PositionList: FC<PLProps> = ({ election }) => {
                       callBackFunc={() => deletePosition(position.id)}
                       isOpen={isDeletePositionModalOpen}
                       closeModal={() => closeDeletePositionModal()}
-                      cleanUpFunc={() => refreshPage()}
+                      cleanUpFunc={() => {
+                        updateElection();
+                      }}
+                      expectedResult={204}
```

Comment:
This is a great step forward.

In the future, for performance, we should try to find a nice functional way to render one of these per (list).

The delete button's onClick can be used to call a function with the given position's ID, which would set some state objects. We could then use a useEffect to watch those and open the modal when they change/are ready.